### PR TITLE
Аdd try/except block when calling get_all method

### PR DESCRIPTION
**Pull request by TomeCirun**
I put `get_all` method into a try/except block...

image

### CODE REVIEW

**Comment by TomeCirun** [comment]
also on this PR i moved the `frequency_options` to their own module `choices.py`, added cli command that shows the available choices and replaced the `render()` method with `redirect()`... this will fix form re-submission on page refresh.

File: ckanext/datastore_refresh/choices.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import ckan.plugins.toolkit as toolkit
+from ckan.common import config
+
+from ckanext.datastore_refresh.schema import default_frequency_options_schema
+
+
+DEFAULT_VALUES = [
+    {"value": "5", "text": "5 minutes"}, 
+    {"value": "120", "text": "2 hours"}, 
+    {"value": "1440", "text": "Dialy"}
```
Comment (tino097) [inline ckanext/datastore_refresh/choices.py:12]: typo -> Dialy to Daily

File: ckanext/datastore_refresh/choices.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+]
+
+def load_options():
+    frequency_options = config.get('ckanext.datastore_refresh.frequency_options')
```
Comment (tino097) [inline ckanext/datastore_refresh/choices.py:16]: change to `frequency_optons_path`

**Comment by tino097** [comment]
@TomeCirun @MarkCalvert i merged two other PR's into this one as this is ready to go to develop

**Review by MarkCalvert — APPROVED** [review]

**Pull request merged**